main/res/layout/actionbar_popup.xml

```diff
@@ -35,8 +35,7 @@
     <ImageView
         android:id="@+id/overflowActionBar"
         style="@style/action_bar_action"
-        android:longClickable="true"
-        android:src="@android:drawable/ic_menu_more" />
-
+        android:longClickable="false"
```

Comment:
This looks unrelated to #7251.

Reply:
You are right.
I will create a separate PR.

Reply:
Ups...to late I guess. Anything regarding the longClick we should now, why it was changed.